test/new-e2e/pkg/e2e/suite.go

```diff
@@ -467,6 +477,9 @@ func (bs *BaseSuite[Env]) SetupSuite() {
 		// `panic()` is required to stop the execution of the test suite. Otherwise `testify.Suite` will keep on running suite tests.
 		panic(err)
 	}
+	if bs.initOnly {
+		bs.T().Skip("INIT_ONLY is set, skipping tests")
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Nice